packages/subgraph/tests/multisig/multisig.test.ts

```diff
@@ -420,8 +424,9 @@ test('Run Multisig (handleMembersRemoved) mappings with mock event', () => {
 
   // checks
   assert.fieldEquals('MultisigApprover', memberId1, 'id', memberId1);
-  assert.notInStore('MultisigApprover', memberId2);
-
+  assert.fieldEquals('MultisigApprover', memberId1, 'isActive', 'true');
+  assert.fieldEquals('MultisigApprover', memberId2, 'id', memberId2);
+  assert.fieldEquals('MultisigApprover', memberId2, 'isActive', 'false');
   clearStore();
 });
```

Comment:
Can you add a test for the null pointer exception?

Reply:
Not sure what kind of test are you refering to. We had a null pointer exception when we removed one member. In this test we are doing exactly that and we check that the member was not removed and instead it's still there but the isActive is set to false. We could add a check for checking that the approvals and the member are still there after removing a member, is that what you meant?